README.md

```diff
@@ -1,222 +1,74 @@
 
-### Configuring and extending
-
-Components are customizable in two ways: *configuration* and *extension*. Configure a component by setting a component’s variables *before* you import the component, and extend by adding/overriding styles or adding new modifiers *after* importing the component. The order is important, because each component will use its default values for variables unless you provide your own. For example, assume you have the following in your main stylesheet:
-
-```scss
-@import 'variables'; // Do this first, so Stencil is aware of your settings.
-@import '../bower_components/mobify-stencil/dist/components/button/button'; // Import Stencil components before yours.
-@import 'my-components/button'; // Add styles to Stencil components after importing them.
-```
-
-The contents of these files might look like this:
-
-```scss
-// _variables.scss
-
-// Configure some variables for Stencil’s button:
-$button-padding: 5px 10px;
-$button-border-radius: 5px;
-```
-
-```scss
-// my-components/_button.scss
-
-// Override or add to a Stencil style:
-.c-button {
-    border: 0; // override
-    background-color: hsl(200, 70%, 50%); // add
-}
-
-// Extend with a new modifier, in this case for a button that uses the site’s brand color:
-.c-button.c--brand-color {
-    background: $brand-color;
-    color: #fff;
-}
-```
-
-
-## Naming conventions
-
-Stencil uses Mobify’s variation on the BEM (Block/Element/Modifier) convention. Read more about the details of the convention in our open source [CSS style guide](https://github.com/mobify/mobify-code-style/blob/master/css/Readme.md#component-oriented-naming).
-
-Stencil adds one extra convention to this: utility classes. These special classes are conceptually a bit like a Sass mixin: they allow you to “mix in” new styles into an existing component. Unlike mixins, Stencil utility classes are applied as classes in your markup. An example of a utility class is `u-width-full`. It does exactly what you’d think: makes an element take up the full width of its container. This utility class can be used in conjunction with the Grid component (along with other width utilities) for building layout, and can also be applied for example to make buttons full-width without needing a custom full-width button modifier.
 
 Some things to note about utility classes:
 
 1. Utility classes should apply a focused set of CSS properties that are applicable across a wide range of components;
-2. Utility classes use a `u-` prefix instead of the `c-` prefix used for a component’s class name;
+2. Utility classes use a `.u-` prefix instead of the `.c-` prefix used for a component’s class name;
 3. Utility classes almost always use `!important` in their styles. This is to ensure their styles always apply on top of other styles. Use utiliy classes surgically to adjust existing components, and use with care.
```

Comment:
This `!important` looks dangerous. It's applied in the markup, this could be a component markup and there won't be opportunity to overwrite it in template css.